migrations/20220117025847_email_data.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TYPE valid_status AS ENUM ('running', 'completed', 'stopped');
+CREATE TABLE blk_vrfy_job (
+    id SERIAL PRIMARY KEY,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    total_records INTEGER NOT NULL,
+    job_status VALID_STATUS NOT NULL DEFAULT VALID_STATUS 'running'
+);
+CREATE TABLE ema_vrfy_rec (
+    job_id INTEGER,
+    email_id TEXT,
+    result JSONB,
+    PRIMARY KEY(job_id, email_id),
```

Comment:
So a job can have duplicate email ids? But yeah I can see why checking that will be tedious I'll drop the `email_id` but I think we should keep `job_id` because it will act as index and make aggregate info queries for the job faster.

Reply:
> So a job can have duplicate email ids

I'm not sure we have the same terminologies, I don't quite understand "email id". But yeah, a job can have duplicate email addresses to check.